### PR TITLE
monitoring: Describe query log file feature for Prometheus

### PR DESCRIPTION
**Pull request by philipgough**
This documents the changes made in https://github.com/openshift/cluster-monitoring-operator/pull/1373 which enables Prometheus query log file https://prometheus.io/docs/guides/query-log/

### CODE REVIEW

File: modules/monitoring-setting-query-log-file-for-prometheus.adoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+----
+$ oc -n openshift-monitoring get pods
+----
++
+
+. Read the query log:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring exec prometheus-k8s-0 cat <path>
```
Comment (sthaha) [inline modules/monitoring-setting-query-log-file-for-prometheus.adoc:72]: syntax for `oc exec` is `oc exec (POD | TYPE/NAME) [-c CONTAINER] [flags] -- COMMAND [args...]` 
hence it is safer to use `--` to separate `command` to be run

**Comment by Preeticp** [comment]
@PhilipGough thank you for the PR, please note that since this PR is for the 4.10 version of docs, we can merge this PR only post the 4.9 GA. Meanwhile, depending on @bburt-rh 's capacity he will start reviewing the PR.

**Comment by PhilipGough** [comment]
/hold

Depends on https://github.com/openshift/cluster-monitoring-operator

File: modules/monitoring-setting-query-log-file-for-prometheus.adoc [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheus:
```
Comment (jan--f) [inline modules/monitoring-setting-query-log-file-for-prometheus.adoc:47]:
```suggestion
    prometheusK8s:
```

File: modules/monitoring-setting-query-log-file-for-prometheus.adoc [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+. Edit the `ConfigMap` object:
+** *To set the query log file for Prometheus in the `openshift-monitoring` project*:
+.. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+.. Add `queryLogFile: <path>` for `prometheus` under `data/config.yaml`:
```
Comment (jan--f) [inline modules/monitoring-setting-query-log-file-for-prometheus.adoc:36]:
```suggestion
.. Add `queryLogFile: <path>` for `prometheusK8s` under `data/config.yaml`:
```

**Review by jan--f — CHANGES_REQUESTED** [review]
A copy/paste error sneaked in.

**Comment by bburt-rh** [comment]
I've created an updated PR for this content at https://github.com/openshift/openshift-docs/pull/43449

**Comment by bburt-rh** [comment]
@PhilipGough This PR can be closed, since the content from https://github.com/openshift/openshift-docs/pull/43449 has already been completed and merged.

**Pull request closed without merging**